Testscripts/Linux/ovs_setup.sh

```diff
@@ -38,17 +38,34 @@ function install_ovs () {
 	esac
 	ssh "${1}" ". ${UTIL_FILE} && install_package ${packages[@]}"
 
+	# For older versions of dpdk get old ovs pair
+	# DPDK releases are with .tar only
+	if [[ $dpdkSrcLink =~ .tar ]];
+	then
+		# check dpdk version
+		ovsOldSrc="https://github.com/openvswitch/ovs/archive/v2.11.2.tar.gz"
```

Comment:
is it a fixed value? we'd better to put it into the param like this https://github.com/LIS/LISAv2/blob/master/XML/TestCases/PerformanceTests-DPDK.xml#L11, and it can be replaced during the run.

Reply:
Please merge if this change request completes.

Reply:
We already have a param but we want to handle it through automation as user/pipeline running automation won't always have any knowledge of which version is paired with which one.
Please let me know if you have any suggestions to implement it.

Reply:
@abhimarathe, If this version 2.11.2 will be permanent, then keep it as-is.